### DISCUSSION TITLE
Light & Dark mode toggle switch

### DISCUSSION THREAD

**Discussion by zerenxyz** [moeyua/astro-theme-typography#79] [Ideas]
Could it be an idea maybe to add a toggle somewhere that fits the Typography theme style to switch between Dark and Light mode or also enable "System Preference ". Not sure if this is something possible to do @moeyua ?

**Comment by moeyua** [comment]
Hi @zerenxyz ,

I think this suggestion is very good, and it doesn't seem too difficult to implement. However, I haven't decided where this button should be placed yet. Do you have any good suggestions? I would be happy to hear your opinion.

**Reply by zerenxyz** [reply]
Hi @moeyua

Hmm, to fit with the style of the theme, it could be maybe placed on the bottom footer section as text Light/Dark like in this screenshot
![image](https://github.com/moeyua/astro-theme-typography/assets/52886201/45470c84-ad65-49bc-a04b-fd12f40de7dd)

Or otherwise, maybe adding a Light/Dark/System Preference icon below/above the social icons might look nice and fitting?
![image](https://github.com/moeyua/astro-theme-typography/assets/52886201/354cf68f-1815-45c6-b07b-7381f17a1e1b)
